### ISSUE TITLE
Load TFJS trained model in java application.

### ISSUE THREAD

**Issue by cmckamey** [tensorflow/java#476]
I have a model that was developed and trained in TFJS and I want to use it in my java application. 

I managed to convert it to keras and then to a SavedModel but I am wondering if there is a way to just load the TFJS model directly and spare the clunk conversion process that involves several additional steps and python.

Is such a thing even possible?

**Comment by Craigacp** [comment]
Looks like you should be able to go directly to a TF Saved Model with the converter, but I don't think the TFJS file format is documented so we couldn't support it.

**Comment by cmckamey** [comment]
Yeah I have it working very well post conversion, just wish I could save that step. Makes implementation of my project tricky.

**Comment by Craigacp** [comment]
Unfortunately not. At the moment it looks like tfjs uses something Keras-like as it's JSON format, but we don't have all the Python bits necessary to load in Keras models that aren't exported in SavedModel format.
